cli/src/main/scala/com/lightbend/rp/reactivecli/argparse/InputArgs.scala

```diff
@@ -219,6 +219,11 @@ object InputArgs {
             .text("Sets the cluster IP for Service resources")
             .action(ServiceArgs.set((v, args) => args.copy(clusterIp = Some(v)))),
 
+          opt[String]("override-version")
+            .text("Uses specified version tag for generated resources instead of version in the docker image")
+            .optional()
+            .action(KubernetesArgs.set((v, args) => args.copy(overrideVersion = Some(v)))),
```

Comment:
Given the above, this will become something like `GenerateDeploymentArgs.set((v, args) => args.copy(version = Some(v)))`